lib/D2L.Extensibility.AuthSdk.IntegrationTests/AnonymousRequestTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Net;
+using D2L.Extensibility.AuthSdk.IntegrationTests.Domain;
+using D2L.Extensibility.AuthSdk.IntegrationTests.Helpers;
+using D2L.Extensibility.AuthSdk.IntegrationTests.Providers;
+using NUnit.Framework;
+
+namespace D2L.Extensibility.AuthSdk.IntegrationTests {
+
+	[TestFixture]
+	public class AnonymousRequestTests {
+
+		private HttpWebRequest m_request;
+
+		[SetUp]
+		public void TestSetup() {
+			m_request = RequestProvider.PrepareApiRequest( ContextProvider.AnonUserContext(), RouteProvider.VersionsRoute );
+		}
+
+		[Test]
+		public void SendAnonymousRequest_ResponseReceived() {
+			Assert.DoesNotThrow( () => { using( HttpWebResponse response = m_request.GetResponse() as HttpWebResponse ) { } } );

```

Comment:
I simplified the using statement and hopefully improved the whitespace a bit with commit 338c23e.